control/DeadbandXboxController.java

```diff
@@ -19,6 +39,23 @@
     public DeadbandXboxController(int port, double deadbandCutoff) {
         super(port);
         this.deadbandCutoff = deadbandCutoff;
+
```

Comment:
Can these be named either according to what the WPILib XboxController values are named, or what we name them in the RapidReact project? Just to make the transition easier?

Reply:
they are all the same except for the joysticks,,, I changed all of the bindings on my local project and it was literally just a matter of adding `xb.` in front of the commands